tests/Endpoints/Users/ContractTest.php

```diff
@@ -69,4 +69,51 @@ public function testCreateContractFees(): void
          */
         self::assertInstanceOf(Ewallet::class, $contract->getEwallet());
     }
+
+    /**
+     * Test get contract fees.
+     *
+     * @return void
+     */
+    public function testGetContractFees(): void
+    {
+        $user = new User(['id' => 'external-user-id']);
+        $ewallet = new Ewallet([]);
+        $contract = new Contract([
+            'action' => 'debit',
+            'currency' => 'AUD',
+            'ewallet' => $ewallet,
+            'group' => 'mastercard',
+            'fixed_fee' => '0.02',
+            'variable_rate' => '0.10',
+            'user' => $user
+        ]);
+
+        self::assertSame('debit', $contract->getAction());
+        self::assertSame('mastercard', $contract->getGroup());
+        self::assertSame('AUD', $contract->getCurrency());
+        self::assertSame($ewallet, $contract->getEwallet());
+        self::assertSame('0.02', $contract->getFixedFee());
+        self::assertSame('0.10', $contract->getVariableRate());
+        self::assertSame($user, $contract->getUser());
+    }
+
+    /**
+     * Ensure uris have correct endpoints.
+     *
+     * @return void
+     */
+    public function testUris(): void
+    {
+        $user = new User([]);
+        $contract = new Contract([
+            'user' => $user
+        ]);
+
+        $createUri = \sprintf('/users/%s/contracts', $user->getId());
+        $getUri = \sprintf('/users/%s/contracts', $user->getId());
+
+        self::assertSame($createUri, $contract->uris()['create'] ?? []);
```

Comment:
`self::assertSame(['create' => '', 'get' => ''], $contract->uris())` can be a better assertion here. Would make sure the test fails if some other endpoint is added and would force for the test to be updated as well.